.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,136 @@
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+          cache: 'npm'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)